android-components/components/service/sync-autofill/src/main/java/mozilla/components/service/sync/autofill/AutofillCrypto.kt

```diff
@@ -51,6 +51,10 @@ class AutofillCrypto(
         key: ManagedKey,
         encryptedCardNumber: CreditCardNumber.Encrypted,
     ): CreditCardNumber.Plaintext? {
+        if (encryptedCardNumber.number.isEmpty()) {
```

Comment:
Could we add a quick test in https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/service/sync-autofill/src/test/java/mozilla/components/service/sync/autofill/AutofillCryptoTest.kt?